aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -58,94 +55,23 @@ spec:
 
   - name: ClusterName
     type: Input
-    description: What is the name of the cluster?
+    description: What do you want to name the cluster?
 
-  - name: NodeAutoScalingGroupDesiredSize
+  - name: Prefix
     type: Input
-    default: 3
-    description: What is the desired number of nodes in the cluster?
-    pattern: "([0-9])+"
-
-  - name: NodeInstanceType
-    type: Select
-    description: What EC2 instance type do you want to use for the nodes?
-    default: t2.medium
-    options:
-    - t2.medium
-    - t2.large
-    - t2.xlarge
-    - t2.2xlarge
-    - m3.medium
-    - m3.large
-    - m3.xlarge
-    - m3.2xlarge
-    - m4.large
-    - m4.xlarge
-    - m4.2xlarge
-    - m4.4xlarge
-    - m4.10xlarge
-    - m5.large
-    - m5.xlarge
-    - m5.2xlarge
-    - m5.4xlarge
-    - m5.12xlarge
-    - m5.24xlarge
-    - c4.large
-    - c4.xlarge
-    - c4.2xlarge
-    - c4.4xlarge
-    - c4.8xlarge
-    - c5.large
-    - c5.xlarge
-    - c5.2xlarge
-    - c5.4xlarge
-    - c5.9xlarge
-    - c5.18xlarge
-    - i3.large
-    - i3.xlarge
-    - i3.2xlarge
-    - i3.4xlarge
-    - i3.8xlarge
-    - i3.16xlarge
-    - r3.xlarge
-    - r3.2xlarge
-    - r3.4xlarge
-    - r3.8xlarge
-    - r4.large
-    - r4.xlarge
-    - r4.2xlarge
-    - r4.4xlarge
-    - r4.8xlarge
-    - r4.16xlarge
-    - x1.16xlarge
-    - x1.32xlarge
-    - p2.xlarge
-    - p2.8xlarge
-    - p2.16xlarge
-    - p3.2xlarge
-    - p3.8xlarge
-    - p3.16xlarge
-
-  - name: ProvisionEFS
-    type: Confirm
-    description: "Do you want to provision Amazon Elastic File System?"
-    default: false
+    description: "Specify the application prefix (use the default if you're not sure):"
+    default: aws-eks-
+    pattern: "([a-z0-9-]+-)+"
 
   files:
-  - path: xebialabs/xld-infra-env.yaml.tmpl
-  - path: xebialabs/xld-cloudformation-apps.yaml.tmpl
-  - path: xebialabs/xlr-pipeline-ci-cd.yaml.tmpl
-  - path: xebialabs/xlr-pipeline-destroy.yaml.tmpl
-  - path: xebialabs/USAGE.md.tmpl
+  - path: xebialabs/aws-basic-eks-cluster.yaml.tmpl
+  - path: xebialabs/aws-basic-eks-cluster-applications.yaml.tmpl
+  - path: xebialabs/aws-basic-eks-cluster-environments.yaml.tmpl
+  - path: xebialabs/USAGE-aws-basic-eks-cluster.md.tmpl
+
   - path: kubernetes/aws-auth-cm.yaml
   - path: cloudformation/cfn-secret-provider.zip
```

Comment:
I don't find this file in the PR?

Reply:
Do you mean `cloudformation/cfn-secret-provider.zip`. I definitely see it

Reply:
Yes, I see it only for the intermediate blueprint not for the basic one. If you search for it there is onely 1 file in the PR